### ISSUE TITLE
Delete Group deletes wrong group

### ISSUE THREAD

**Issue by Walsonde** [fastaddons/GroupSpeedDial#166]
I secondary clicked on a group and selected 'delete group' from the appearing menu. After the confirmation prompt the group left to the one I clicked was deleted. The group that I secondary clicked on is still there.

I will not (yet) try to reproduce the bug before I saved all my dials as firefox bookmarks, since I have some dials that are really important to me (as were the ones just deleted). 

**Device info:**
 - operating system: Linux Mint 21 Vanessa
 - Browser and version: Firefox 105.0.2
 - GroupSpeedDial version: v21.8

**Comment by fastaddons** [comment]
Hello,
Are you sure you've clicked the correct group?
I can't reproduce it here.
Also, you can undo any recent change from the main menu / History backups. So even deleting a group can be undone easily.

**Comment by Walsonde** [comment]
I found the cause. Sorry to have bothered you. (._.)

I was using the desktop client of a cloud service at the time. This client had a bug, which led to .config- and other system files being replaced by older versions from the cloud instead of vice versa. My complete Linux setup and all the software I was using consecutively was corrupted more and more, resulting in various different faulty behaviours.

I set up my computer new from scratch to make sure, there's no bug from the cloud client left and did not encounter any problems ever since. If the problem with the accidentally deleted group hadn't been the first occurence in a long list of malfunctions, I would not have suspected GroupSpeedDial in the first place.

I hope you will accept my apology.

Stay safe!
